csc2/macc_so.c

```diff
@@ -3125,10 +3140,18 @@ static int dyns_get_field_option_comn(char *tag, int fidx, int option,
             case CLIENT_BYTEARRAY: {
                 int *bytes;
                 int length;
+                if (*value_type == CLIENT_BYTEARRAY && vbsz >= tables[tidx].sym[fidx].szof && 
+                   tables[tidx].sym[fidx].szof == 16 && 
+                   strcasecmp(tables[tidx].sym[fidx].fopts[i].value.strval, "GUID()") == 0) {
```

Comment:
May be a comment that explains why we look for "GUID()" here and for "(GUID())" (in parens) above.

Reply:
this was deleted.